source/_includes/navigation.html

```diff
@@ -28,7 +28,7 @@
 		<li>
 			<a href="#">Posts</a>
 			<ul class="dl-submenu">
-				<li><a href="/posts/">All Posts</a></li>
+				<li><a href="/archives/">Archives</a></li>
```

Comment:
Btw, I just realised there's a tiny issue my implementation. The footer is inside the last container.  I'll fix it.